docs/developer/other/IDE_INTEGRATION.md

```diff
@@ -0,0 +1,31 @@
+# IDE Integration
+
```

Comment:
Good question.

Also -- does this content document anything that covered in very superficial Idea and/or Eclipse documentation?  Does it document anything that Idea and/or Eclipse users would be unlikely to know?